tsconfig.build.json

```
@@ -0,0 +1,12 @@
+{
+  "extends": "./tsconfig.json",
+  "compilerOptions": {
+    "declaration": true,
+    "declarationDir": "dist",
+    "noEmit": false,
+    "outDir": "dist",
+    "rootDir": "src"
+  },
+  "include": ["./src/**/*"],
+  "exclude": ["./src/**/*.test.ts", "./src/**/*.test.js"]
```

No more JS tests to exclude
```suggestion
  "exclude": ["./src/**/*.test.ts"]
```

---

Yup! Copied the latest version from the module template in [f045643](https://github.com/MetaMask/eth-json-rpc-infura/pull/58/commits/f04564357b90a56f7c371e7b881246abb37fee44).